.github/workflows/hvd-tests.yml

```diff
@@ -72,6 +72,10 @@ jobs:
         with:
           target_dir: /tmp
 
+      - name: Check if Horovod actually exists
+        shell: python
```

Comment:
Here you should keep `shell: bash -l {0}` otherwise conda env wont be activated.
Try to run `python -c "import horovod.torch as hvd"` to see what it tells